Header header

int32 arduinoMillis
int32 batteryMillivolts
int32 angleY
int32 angleX
int32 angleZ
int32 driveLeft
int32 driveRight
int32 speedLeft
int32 speedRight
int32 distanceLeft
int32 distanceRight
int32 lineSensor0
int32 lineSensor1
int32 lineSensor2
int32 lineSensor3
int32 lineSensor4
int32 irInput
